src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -688,4 +696,17 @@ private long getLatestSequenceNum(TableId tableId) throws InterruptedException,
       throw e;
     }
   }
+
+  public static String normalize(String name) {
```

Comment:
this should also be used when handling DDL and committing a batch of DML events. Or is that going to be in a follow up PR?